### DISCUSSION TITLE
fpm-search plugin clone

### DISCUSSION THREAD

**Discussion by urbanjost** [fortran-lang/fpm#661] [General]
I cannot seem to contact @brocolis. I hope all is well.  His github site is gone. I tried to reconstruct the fpm-search utility. If it is OK with Carlos and he no longer is interested in providing them I am hoping his projects can be hosted elsewhere. As regards this site, perhaps fpm-search should be hosted here?  So far there has not been a huge growth in the registry but if that changes having a tool for searching it will be invaluable.  It is not too hard to browse the registry page at this point, but if you optimistically hope for something more like other language's registry sorting though tens of thousands of packages without such a tool is usually very unpleasant; albeit if some of those registries had better web interfaces it would not have to be so tedious :>.  I hope he is just moving things to gitlab or something similar, but I have not been able to contact him for quite some time :(  . I was actually hoping the registry would be growing rapidly, actually expected it to.  I think a line-mode tool for submitting code as a candidate for the registry would help some; there are quite a few steps that require installing infrastructure and using potentially unfamiliar materials currently required. Something on github using CI/CD scripts might make that less daunting.  If anyone knows what is up I'd like to know, as appropriate.

It would really be nice if registered packages were archived too, by the way.

**Comment by vmagnin** [comment]
@urbanjost 
I found its GitLab page in my search engine, looking for its cairo-fortran library: https://gitlab.com/brocolis/
but it has also disappeared! :-(
Maybe he has changed his username?

On this page, there is a list of his deleted repositories:
https://hao0452.net/brocolis

On my side, I have just on my hard-disk a clone of the cairo-fortran project dated from the 6th April 2021, probably not the last version.

**Reply by awvwgk** [reply]
For what it is worth, most of his projects were moved to sourcehut (see https://git.sr.ht/~une/cairo-fortran).

**Comment by vmagnin** [comment]
Thanks @awvwgk 
I have also found:
https://git.sr.ht/~une/c-util
https://git.sr.ht/~une/dynload-julia
https://git.sr.ht/~une/dynload-base
But no fpm-search...

They can be cloned without logging in using the read-only https links. Curiously, if I click on the root [~une](https://git.sr.ht/~une/), it is said _"This user has no repositories"._ Maybe because I am not logged in?
